export const NotFoundByIdError = (field: string): string => {
  return `Ao tentar buscar um(a) ${field}, n√£o foi encontrado!`
}
